src/Common/constants.tsx

```diff
@@ -343,6 +343,7 @@ export const ADMITTED_TO = [
   { id: "2", text: "ICU" },
   { id: "6", text: "Bed with oxygen support" },
   { id: "7", text: "Regular" },
+  { id: "None", text: "None" },
```

Comment:
```suggestion
  { id: "None", text: "No bed assigned" },
```

Reply:
Just saw this comment :sweat_smile: . Thanks for making the changes! :blush: